webpack.config.js

```diff
@@ -0,0 +1,432 @@
+// It requires a trailing slash, or the file assets will get an incorrect path.
+const publicPath = '/'
+
+// Some apps do not use client-side routing with pushState.
+// For these, "homepage" can be set to "." to enable relative asset paths.
+const shouldUseRelativeAssetPaths = publicPath === './'
+
+// `publicUrl` is just like `publicPath`, but we will provide it to our app
+// as %PUBLIC_URL% in `index.html` and `process.env.PUBLIC_URL` in JavaScript.
+// Omit trailing slash as %PUBLIC_URL%/xyz looks better than %PUBLIC_URL%xyz.
+const publicUrl = publicPath.slice(0, -1)
+
+// replace globalObject when we are in development so HMR and web workers would work together
+// for details see:
+// https://github.com/webpack/webpack/issues/6642 (closed in favor of 6525)
+// https://github.com/webpack/webpack/issues/6525
+const globalObject = isEnvDevelopment ? 'this' : undefined
+
+// This is the production configuration.
+// It compiles slowly and is focused on producing a fast and minimal bundle.
+// The development configuration is different and lives in a separate file.
+module.exports = {
+  mode: isEnvProduction ? 'production' : isEnvDevelopment && 'development',
+  bail: true,
+  devtool: isEnvProduction ? 'source-map' : 'cheap-module-source-map',
+  entry: [
+    // require.resolve('./config/polyfills'),
+    isEnvDevelopment && require.resolve('react-dev-utils/webpackHotDevClient'),
+    PATHS.INDEX_JS,
+  ].filter(Boolean),
+
+  output: {
+    path: PATHS.OUTPUT,
+    pathinfo: isEnvDevelopment,
+    filename: 'static/js/[name].[hash:8].js',
+    chunkFilename: 'static/js/[name].[chunkhash:8].chunk.js',
+    publicPath,
+    globalObject,
+    devtoolModuleFilenameTemplate: isEnvDevelopment
+      ? info => path.resolve(info.absoluteResourcePath).replace(/\\/g, '/')
+      : info =>
+        path
+          .relative(PATHS.SOURCE, info.absoluteResourcePath)
+          .replace(/\\/g, '/'),
+  },
+
+  resolve: {
+    modules: [
+      PATHS.SOURCE,
+      'node_modules',
+    ],
+    extensions: ['.js', '.jsx', '.json'],
+    plugins: [
+      // Prevents users from importing files from outside of src/ (or node_modules/).
+      // This often causes confusion because we only process files within src/ with babel.
+      // To fix this, we prevent you from importing files out of src/ -- if you'd like to,
+      // please link the files into your node_modules/ and let module-resolution kick in.
+      // Make sure your source files are compiled, as they will not be processed in any way.
+      new ModuleScopePlugin(
+        PATHS.SOURCE,
+        ['package.json']
+      ),
+    ],
+  },
+
+  module: {
+    strictExportPresence: true,
+
+    rules: [
+      // Disable require.ensure as it's not a standard language feature.
+      { parser: { requireEnsure: false } },
+
+      // First, run the linter.
+      // It's important to do this before Babel processes the JS.
+      {
```

Comment:
I sggest to disable eslint formatter, because we have husky + eslint and this is not nesessuary

Reply:
Fixed. We show errors only in production now.